File: src/main/java/superstra/liquidhoney/LiquidHoney.java
package superstra.liquidhoney;

import net.fabricmc.api.ModInitializer;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class LiquidHoney implements ModInitializer {
	public static final String MOD_ID = "liquid-honey";
	// This logger is used to write text to the console and the log file.
	// It is considered best practice to use your mod id as the logger's name.
	// That way, it's clear which mod wrote info, warnings, and errors.
    public static final Logger LOGGER = LoggerFactory.getLogger("liquid-honey");

	@Override
	public void onInitialize() {
		// This code runs as soon as Minecraft is in a mod-load-ready state.
		// However, some things (like resources) may still be uninitialized.
		// Proceed with mild caution.

		LOGGER.info("Hello Fabric world!");
	}
}

File: src/main/kotlin/com/superstra/liquidhoney/LiquidHoney.kt
package com.superstra.liquidhoney

import com.superstra.liquidhoney.block.ModBlocks
import com.superstra.liquidhoney.item.ModItems
import net.fabricmc.api.ModInitializer
import org.slf4j.Logger
import org.slf4j.LoggerFactory

object LiquidHoney : ModInitializer {
	val MOD_ID = "liquid-honey"
    val LOGGER: Logger = LoggerFactory.getLogger(MOD_ID)

	override fun onInitialize() {
		// This code runs as soon as Minecraft is in a mod-load-ready state.
		// However, some things (like resources) may still be uninitialized.
		// Proceed with mild caution.
		LOGGER.info("Hello Fabric world!")

		ModItems.registerModItems()
		ModBlocks.registerModBlocks()
	}
}